Misc/NEWS.d/next/Library/2020-05-26-21-25-34.bpo-40767.jw964B.rst

```diff
@@ -0,0 +1,4 @@
+Pure Wayland environments (without xwayland) do not have :exc:`DISPLAY`
+environment variable set. Instead they have :exc:`WAYLAND_DISPLAY`.  The
+:exc:`webbrowser` API now properly fetches default browser in such
```

Comment:
I think `:exc:` is used for exception, the entry could also be less verbose. What do you think of:

```
:mod:`webbrowser` now properly find the default browser in Wayland by checking the `WAYLAND_DISPLAY` environment variable. Patch contributed by Jérémy Attali.
```

?

Reply:
Thanks @remilapeyre I just pushed an updated blurb.